robot/impl/robot_options.go

```diff
@@ -19,6 +19,7 @@ type options struct {
 	// shutdownCallback provides a callback for the robot to be able to shut itself down.
 	shutdownCallback func()
 
+	// whether FTDC is enabled
```

Comment:
Did the linter complain? Because I really don't want to nitpick that the tense of `enabled` implies the member is describing what is currently running -- as opposed simply describing the users intent for when the robot is started!

Reply:
No idea why I added this (was not linter complaining); switched to `whether or not to run FTDC`. Hopefully that captures what you're getting at here, but let me know if not.